The upper channel limit in North America is 36
No need to waste time scanning all the way to channel 69
--- libhdhomerun/hdhomerun_channels.c
+++ libhdhomerun/hdhomerun_channels.c
@@ -95,7 +95,7 @@
 	{  2,   4,  57000000, 6000000},
 	{  5,   6,  79000000, 6000000},
 	{  7,  13, 177000000, 6000000},
-	{ 14,  69, 473000000, 6000000},
+	{ 14,  36, 473000000, 6000000},
 	{  0,   0,         0,       0}
 };
 
